product_state/data/product_state_data.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+  <data>
+    <!-- States -->
+    <record id="product_state_draft" model="product.state">
+      <field name="code">draft</field>
+      <field name="name">In Development</field>
+      <field name="sequence">1</field>
+    </record>
+    <record id="product_state_sellable" model="product.state">
+      <field name="code">sellable</field>
+      <field name="name">Normal</field>
+      <field name="sequence">2</field>
+    </record>
+    <record id="product_state_end" model="product.state">
+      <field name="code">end</field>
+      <field name="name">End of Lifecycle</field>
+      <field name="sequence">3</field>
+    </record>
+    <record id="product_state_obsolete" model="product.state">
+      <field name="code">obsolete</field>
+      <field name="name">Obsolete</field>
+      <field name="sequence">4</field>
```

Comment:
Change sequence value 1 -> 10, 2 -> 20 ...
It allow create new product states without change default dat (between draft and sellable as example)